src/main/java/teammates/ui/controller/InstructorFeedbackEditCopyPageData.java

```diff
@@ -7,6 +7,8 @@
 
 public class InstructorFeedbackEditCopyPageData extends PageData {
     public List<CourseAttributes> courses;
+    public String courseId;
+    public String fsName;
```

Comment:
Are these accessed directly outside this class? We can declare them as private and use get methods instead. As far as possible, we can try to make attributes private instead of public.